server/sonar-process-monitor/src/main/java/org/sonar/application/process/ProcessEventListener.java

```diff
@@ -17,23 +17,27 @@
  * along with this program; if not, write to the Free Software Foundation,
  * Inc., 51 Franklin Street, Fifth Floor, Boston, MA  02110-1301, USA.
  */
-package org.sonar.process.monitor;
+package org.sonar.application.process;
 
-import org.junit.Test;
-import org.mockito.Mockito;
+import org.sonar.process.ProcessId;
 
-import static org.mockito.Mockito.*;
+@FunctionalInterface
+public interface ProcessEventListener {
```

Comment:
![MAJOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-major.png 'Severity: MAJOR') Make this interface extend "java.util.function.BiConsumer<ProcessId,ProcessEventListener$Type>" and remove the functional method declaration. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](https://next.sonarqube.com/sonarqube/coding_rules#rule_key=squid%3AS1711)